gradoop-common/src/test/java/org/gradoop/common/model/impl/pojo/ElementTest.java

```diff
@@ -0,0 +1,143 @@
+package org.gradoop.common.model.impl.pojo;
+
+import org.gradoop.common.model.impl.id.GradoopId;
+import org.gradoop.common.model.impl.properties.Properties;
+import org.junit.Test;
+import org.mockito.Mockito;
+import static junit.framework.TestCase.fail;
```

Comment:
This is an internal function AFAIK, use `Assert.fail` instead.

Reply:
done